recipes/ecflow/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "ecFlow" %}
+{% set version = "5.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-Source.tar.gz
+  url: https://confluence.ecmwf.int/download/attachments/8650755/{{ name }}-{{ version }}-Source.tar.gz
+  sha256: 877ce1e3ee818241cb5d8c9cadb7330beb8d24af331d047e87ad937042fde09d
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxfixes') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxcb') }}  # [linux]
+    - {{ cdt('libxext') }}  # [linux]
+    - {{ cdt('libxau') }}  # [linux]
+  host:
+    - python
+    - boost
+    - qt
+    - xorg-libxfixes  # [linux]
+  run:
+    - python
+    - qt
+
+test:
+  commands:
+    - conda inspect linkages -p $PREFIX ecflow
+    - conda inspect objects -p $PREFIX ecflow  # [osx]
+    - ecflow_client --help
+    - ecflow_server --help
+    - ecflow_standalone --help
+    - ecflow_ui_log -h
+
+about:
+  home: https://confluence.ecmwf.int/display/ECFLOW
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: Apache 2.0
```

For consistency with other recipes:
```suggestion
  license: Apache-2.0
```